Source/App/AppController+Push.swift

```diff
@@ -132,7 +132,6 @@ extension AppController {
                 
             }
         }
-        Analytics.shared.updatePushToken(pushToken: token)
```

Comment:
Is there a reason we aren't tracking this here anymore?

Reply:
Mixpanel allowed sending the push token so we could send remote push notifications directly from Mixpanel, for instance, for marketing purposes (to let the user know there was a new update, or communicate new features).
Posthog doesn't provide this feature. It can automatically record push notifications, including sending the device token inside an event, but I disabled it as there is no use in storing this information in Posthog.